tests/lib/rules/prefer-ember-test-helpers.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/prefer-ember-test-helpers');
+const RuleTester = require('eslint').RuleTester;
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+});
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const TEST_FILE_NAME = 'some-test.js';
+
+ruleTester.run('prefer-ember-test-helpers', rule, {
+  valid: [
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { blur } from '@ember/test-helpers';
+
+      test('foo', async (assert) => {
+        await blur();
+      });`,
+    },
+
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { find } from '@ember/test-helpers';
+
+      test('foo', async (assert) => {
+        await find();
+      });`,
+    },
+
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { focus } from '@ember/test-helpers';
+
+      test('foo', async (assert) => {
+        await focus();
+      });`,
+    },
+  ],
```

Comment:
Let's add valid test cases to ensure we avoid false positives with the wrong function call:

1.
```
{
  filename: TEST_FILE_NAME,
  code: `import { focus } from '@ember/test-helpers';
  test('foo', async (assert) => {
    await focus.wrongFunction();
  });`,
},
```

2.
```
{
  filename: TEST_FILE_NAME,
  code: `import { focus } from '@ember/test-helpers';
  test('foo', async (assert) => {
    await WrongObject.focus();
  });`,
},
```

Reply:
👍

Reply:
To double confirm - should these snippets be missing the import?

Reply:
Yeah the import statement should be included with these ones. I know that `focus.wrongFunction()` is not a valid usage, but these test cases are specifically designed to ensure we only trigger on the right function call, to avoid false positives.

Reply:
Got it!